multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/service/ObjectStoreServiceInfoCreator.java

```diff
@@ -42,10 +44,17 @@ private ObjectStoreServiceInfo createServiceInfoForAzure(Map<String, Object> cre
         String accountName = (String) credentials.get("account_name");
         String sasToken = (String) credentials.get("sas_token");
         String containerName = (String) credentials.get("container_name");
+        URL containerUri;
+        try {
+            containerUri = new URL((String) credentials.get("container_uri"));
+        } catch (MalformedURLException e) {
+            throw new IllegalStateException("Cannot parse container_uri of objectstore", e);
+        }
         return ImmutableObjectStoreServiceInfo.builder()
                                               .provider("azureblob")
                                               .identity(accountName)
                                               .credential(sasToken)
+                                              .endpoint(containerUri.getProtocol() + "://" + containerUri.getHost())
```

Comment:
Can we use `containerUri.toString()`?

Reply:
Seems not, because in the `container_uri` that we have in test environment, URI contains also the path in the end, which would be appended in  `toString()`.
I did not find any validation in Jclouds that strips down the whole URI into only schema+host